jstreegrid.js

```diff
@@ -534,6 +534,9 @@
 				col = this.midWrapper.children("div.jstree-grid-column-"+i);
 				last = $("<div></div>").css(conf).addClass("jstree-grid-div-"+this.uniq+"-"+i+" "+(tr?"ui-widget-header ":"")+" jstree-grid-header jstree-grid-header-cell jstree-grid-header-"+classAdd+" "+cl+" "+ccl).html(val);
 				last.addClass((tr?"ui-widget-header ":"")+"jstree-grid-header jstree-grid-header-"+classAdd);
+				if (col.ellipsis !== false){
```

Comment:
Following the comments on the issue:

````javascript
if (col.ellipsis === true || (col.ellipsis === undefined && this.settings.themes.ellipsis === true)) {
````

(I might have the above wrong in terms of retrieving the setting, but the idea)